internal/core/application/service.go

```diff
@@ -1200,15 +1200,16 @@ func (s *Service) isInitializedAndUnlocked(ctx context.Context) error {
 	return nil
 }
 
-func (s *Service) boltzRefundSwap(swapId, refundTx string) (string, error) {
+func (s *Service) boltzRefundSwap(swapId, refundTx, checkpointTx string) (string, string, error) {
```

Comment:
```suggestion
func (s *Service) boltzRefundSwap(swapId, refundTx, checkpointTx string) (*psbt.Packet, *psbt.Packet, error) {
```

nit: we can move deserialization of psbt here

Reply:
done